source/common/access_log/access_log_formatter.cc

```diff
@@ -301,6 +301,16 @@ RequestInfoFormatter::RequestInfoFormatter(const std::string& field_name) {
       return RequestInfo::Utility::formatDownstreamAddressNoPort(
           *request_info.downstreamRemoteAddress());
     };
+  } else if (field_name == "REQUESTED_SERVER_NAME") {
+    field_extractor_ = [](const RequestInfo::RequestInfo& request_info) {
+      absl::string_view requested_server_name;
+      if (nullptr != request_info.requestedServerName()) {
```

Comment:
This seems odd to me. I think `!request_info.requestedServerName().empty()` will tell you if the requested server name is set.